iOS/ExposureNotification/source/ExposureNotification.csproj

```
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <TargetFrameworks>Xamarin.iOS10</TargetFrameworks>
+    <IsBindingProject>true</IsBindingProject>
+    <RootNamespace>ExposureNotification</RootNamespace>
+    <AssemblyName>Xamarin.iOS.ExposureNotification</AssemblyName>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <FileVersion>1.0.0.0</FileVersion>
+    <IPhoneResourcePrefix>Resources</IPhoneResourcePrefix>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ProcessEnums>true</ProcessEnums>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Xamarin.iOS.ExposureNotification</PackageId>
+    <Title>ExposureNotification iOS APIs</Title>
+    <Summary>C# bindings for Apple ExposureNotification Framework</Summary>
+    <Description>C# bindings for Apple ExposureNotification Framework</Description>
+    <Authors>Microsoft</Authors>
+    <Owners>Microsoft</Owners>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=2128732</PackageProjectUrl>
+    <PackageLicenseFile>License.md</PackageLicenseFile>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackageVersion>1.0.0.0-beta1</PackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Xamarin.iOS" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="Xamarin.iOS.ExposureNotification.targets" Pack="True" PackagePath="build/Xamarin.iOS.ExposureNotification.targets" />
+    <None Include="Xamarin.iOS.ExposureNotification.targets" Pack="True" PackagePath="buildTransitive/Xamarin.iOS.ExposureNotification.targets" />
```

I read of this trick on Twitter... Maybe we give it a go?

```suggestion
    <None Include="Xamarin.iOS.ExposureNotification.targets" Pack="True" PackagePath="build;buildTransitive" />
```